src/libraries/Common/src/System/Marvin.cs

```diff
@@ -108,12 +108,16 @@ private static uint _rotl(uint value, int shift)
 
         private static ulong GenerateSeed()
         {
+            var bytes = new byte[sizeof(ulong)];
+#if NETCOREAPP || NETSTANDARD2_1
+            RandomNumberGenerator.Fill(bytes);
+#else
             using (RandomNumberGenerator rng = RandomNumberGenerator.Create())
             {
-                var bytes = new byte[sizeof(ulong)];
```

Comment:
does it make sense or is it allowed to use stackalloc here?

Reply:
I don't actually know. It's not something I've used myself.

Reply:
This would imply using unsafe which the devs try to avoid if possible. But the files have some small allocations which could be changed to stackalloc tbh

Reply:
> This would imply using unsafe 

It doesn't. The Fill method takes a span, and stackalloc'ing into a span doesn't require unsafe. We should stackalloc here instead of allocating.

Reply:
So I guess the only relevant change actually in this PR is the changes in this file, plus changing it to a `stackalloc`?